init_handler.py

```diff
@@ -125,7 +125,7 @@ def init():
     create_directory(lucidum_dir)
     copy_file(os.path.join("resources", "docker-compose_env_file"), os.path.join(lucidum_dir, ".env"))
     copy_file(os.path.join("resources", "docker-compose_file"), os.path.join(lucidum_dir, "docker-compose.yml"))
-    copy_file(os.path.join("resources", "airflow_docker_job_file"), os.path.join(lucidum_dir + "/airflow/dags/", "airflow_docker.py"))
+    copy_file(os.path.join("resources", "airflow_docker_job_file"), os.path.join(lucidum_dir, "airflow", "dags", "airflow_docker.py"))
```

Comment:
@lucidumio do we want to change to source s3://lucidum-repository/{{customer_name}}/main_external/ ?